Assets/Scripts/Game/PlayerActivate.cs

```diff
@@ -1400,7 +1400,7 @@ public void ChangeInteractionMode(PlayerActivateModes newMode)
             }
 
             // Present new mode to player
-            DaggerfallUI.SetMidScreenText(TextManager.Instance.GetLocalizedText("interactionIsNowInMode").Replace("%s", modeText));
+            if(!noText) DaggerfallUI.SetMidScreenText(TextManager.Instance.GetLocalizedText("interactionIsNowInMode").Replace("%s", modeText));
```

Comment:
Could you put the `DaggerfallUI.SetMidScreenText(...)` on the next line (with proper indentation)?

Reply:
done :)